hawkular-datamining-api/src/test/java/org/hawkular/datamining/api/SerializationTest.java

```diff
@@ -46,7 +47,9 @@ public SerializationTest() {
     @Test
     public void testSubscription() throws IOException {
 
-        DataMiningForecaster forecaster = new DataMiningForecaster(ImmutableMetricContext.getDefault());
+        DataMiningForecaster forecaster = new DataMiningForecaster(new Metric("tenant", "feed", "foo", 1L, 2L, new
```

Comment:
It should be supported, however it more depends on the query for getting metrics in InventoryStorage.